core/Makefile

```diff
@@ -61,6 +61,8 @@ C_INCLUDES += -include stm32h7xx.h
 #ASM_SOURCES += \
 #$(SYSTEM_FILES_DIR)/startup_stm32h750xx.s
 
+BOOT_BIN = $(wildcard $(SYSTEM_FILES_DIR)/dsy_bootloader*)
```

Comment:
Let's make this conditionally assigned with `?=` instead.

That'll make testing new version of the bootloader, or relying on older versions of the bootloader a bit easier in the future.